xchange-stream-binance/src/main/java/info/bitrich/xchangestream/binance/BinanceStreamingMarketDataService.java

```diff
@@ -356,30 +342,67 @@ private OrderbookSubscription connectOrderBook(CurrencyPair currencyPair) {
     return observable;
   }
 
-  private BinanceWebsocketTransaction<TickerBinanceWebsocketTransaction> tickerTransaction(
-      JsonNode node) {
+  private BinanceWebsocketTransaction<TickerBinanceWebsocketTransaction> tickerTransaction(JsonNode node) {
```

Comment:
not necessary but if reducing code/refactoring may want to create a method 

```
  private <T> BinanceWebsocketTransaction<T> readTransaction(JsonNode node, JavaType type, String transactionType) {
    try {
      return mapper.readValue(mapper.treeAsTokens(node), type);
    } catch (IOException e) {
      throw new ExchangeException(String.format("Unable to parse %s transaction", transactionType), e);
    }
  }
```

and call it

`this.<TickerBinanceWebsocketTransaction>readTransaction(s, TICKER_TYPE, "ticker")`

instead of fairly redundant methods